src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -74,8 +74,7 @@ export default function Consume({
     if (!data || !assetTimeout || data.tokenOrders.length === 0) return
 
     const lastOrder = data.tokenOrders[0]
-
-    if (assetTimeout === 'Forever') {
+    if (assetTimeout === 'Forever' || assetTimeout === '0') {
```

Comment:
This should work only with this check: `assetTimeout === '0'` because `assetTimeout` is in seconds

Reply:
Yes, it does. Not sure why the checking was done using `Forever` from the beginning and hope removing it doesn't affect anything else we did not observed yet.

Reply:
Before we were converting to string on init but than it was changed to fix the timeout error and this part somehow was missed